### PR TITLE
CLN: remove unused imports in tools, sm.tsa

### PR DESCRIPTION
**Pull request by jbrockmendel**
Avoids results files that _might_ be auto-generated pending longer-term solution.

Avoids statespace which has other outstanding PRs.

Avoids a couple other files that need more in-depth excavation.

### CODE REVIEW

**Comment by jbrockmendel** [comment]
@josef-pkt deleting unused imports, could easily be delegated

**Comment by jbrockmendel** [comment]
@josef-pkt rebased; merge conflicts and ensuing rebase cycles could be avoided with speedy turnaround (via delegation)

**Comment by jbrockmendel** [comment]
@ChadFulton broadly construed, your domain

**Comment by bashtage** [comment]
Also needs rebase.

File: statsmodels/tsa/api.py [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+           "adfuller", "kpss", "bds",
+           "datetools",
+           "seasonal_decompose",
+           "graphics",
+           "x13_arima_select_order", "x13_arima_analysis",
+           "statespace",
+           "SARIMAX", "UnobservedComponents", "VARMAX", "DynamicFactor",
+           "MarkovRegression", "MarkovAutoregression",
+           "ExponentialSmoothing", "SimpleExpSmoothing", "Holt",
+           "arma_generate_sample", "ArmaProcess"]
```
Comment (bashtage) [inline statsmodels/tsa/api.py]: This is missing innovations, and possibly others now unfortunately.  Could you double check the __all__ in this PR, then it is ready.

**Pull request merged**